sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -35,12 +34,18 @@ namespace Azure.Messaging.EventHubs.Producer
     ///
     public class EventHubProducerClient : IAsyncDisposable
     {
+        /// <summary>The maximum number of attempts that may be made to get a <see cref="TransportProducer"/> from the pool.</summary>
+        internal const int MaximumCreateProducerAttempts = 3;
```

Comment:
I'd prefer to go `private` unless there's a need beyond testing.

Reply:
> I'd prefer to go `private` unless there's a need beyond testing.

You betcha, I used it in some tests to check the ```RetryLogic``` did try the specified number of times.

Reply:
@jsquire can I resolve this issue?

Reply:
I'm still seeing it as `internal`,  I was using it to remind me to double-check....but I've no complaint if you want to mark it resolved.  Feel free to push the button at your discretion.

Reply:
Ok, no worries, I will keep it open.